libs/fluid-elements/tab-group/src/lib/tab/tab.ts

```diff
@@ -99,45 +99,77 @@ export class FluidTab extends LitElement {
     return this._active;
   }
   set active(active: boolean) {
+    const oldActive = this._active;
     // Only set active true if not disabled
     this._active = this.disabled === false ? active : false;
-    // TODO: Figure out why using the old value does not work. The attribute vanishes in the dom/is not set
-    this.requestUpdate('active');
+    this.requestUpdate('active', oldActive);
     this.tabindex = this.active ? 0 : -1;
   }
   private _active = false;
 
-  /** Dispatches the custom event  */
+  /** Defines whether the user focused an element by tabbing or not */
+  @property({ type: Boolean, reflect: false })
+  get tabbed(): boolean {
+    return this._tabbed;
+  }
+  set tabbed(tabbed: boolean) {
+    const oldTabbed = this.tabbed;
+    this._tabbed = tabbed;
+    this.requestUpdate('tabbed', oldTabbed);
+    this.tabindex = tabbed === true ? 0 : -1;
+  }
+  private _tabbed = false;
+
+  /** Dispatches the custom event with the tabid of the clicked tab  */
   private dispatchActiveTabEvent(): void {
     this.dispatchEvent(new FluidTabActivatedEvent(this.tabid));
   }
 
-  /** Handles the click event */
+  /** Handles the click event. Dispatches the tab when a new tab was clicked */
   private handleClick(): void {
     if (!this._active) {
       this.dispatchActiveTabEvent();
     }
   }
 
+  /** Called when the tab loses focus. Removes the focus outline and sets the tab up for TAB navigation. */
+  private handleBlur(): void {
+    this.tabbed = false;
+    this.tabindex = 0;
+  }
+
   /**
    * Render function of the custom element. It is called when one of the
    * observedProperties (annotated with @property) changes.
    */
   render(): TemplateResult {
     const classes = {
       'fluid-tab': true,
-      'fluid-state--active': this._active,
+      'fluid-tabbed': this.tabbed,
+      'fluid-tab-state--active': this._active,
     };
 
     // Linebreak causes the element to have a space
     return html`<span
       class=${classMap(classes)}
+      tabindex=${this.tabindex}
       ?disabled="${this.disabled}"
       @click="${this.handleClick}"
+      @blur="${this.handleBlur}"
     >
       <slot></slot>
     </span>`;
   }
+
+  /** return the span element of the template as HTMLSpanElement */
+  private getSpanElement(): HTMLSpanElement {
+    return this.shadowRoot?.querySelector('span')!;
+  }
+
+  /** Focuses the span element in the template */
+  focus(): void {
+    this.getSpanElement().focus();
```

Comment:
You don't need to query this one every time you call focus. Queries on the dom could take a bit longer, and as the span element never changes, you could save it into a member. 

```ts
private _rootElement: HTMLSpanElement;

firstUpdated(props: Map<string | number | symbol, unknown>): void {
  this._rootElement = this.shadowRoot?.querySelector('span');
}
```

```suggestion
    this._rootElement.focus();
```